src/components/avatar/avatar.visualspec.js

```diff
@@ -0,0 +1,13 @@
+import { percySnapshot } from '@percy/puppeteer';
+import { widths } from '../../../test/percy/widths';
+
+describe('Avatar', () => {
+  beforeAll(async () => {
+    await page.goto('http://localhost:3001/avatar');
+  });
+
+  it('Avatar', async () => {
+    await expect(page).toMatch('when gravatar hash is known');
```

Comment:
I just like having these expect's because if your route isn't loaded, the test fails. Else the tests will pass even if the route doesn't exist... Would be nice actually to have titles or something for each snapshot and display those.